docs/faq.rst

```diff
@@ -115,6 +122,34 @@ See above.
 
 ------
 
+**Q. Why do I see error code 503 with the message “Error: configuration operation in progress on device, Please try again in 2 min” on the 'Pending tasks' table of the app?**
+
+If a BIG-IP is already processing an AS3 asynchronous task, it displays this message for any successive async operations. Once the asynchronous operation is completed, it is possible to perform the next operation like create/update/delete partition or application. Please wait for a few minutes and try the configuration again. 
+
+------
+
+**Q. For all L4-L7 App Services operations, why do I see the message “BIG-IP is processing the request. Please click the 'Pending Tasks' icon to check the status of the pending request.”?**
+
+All the L4-L7 App Services are processed in an asynchronous manner in the background. Hence this message is displayed for every operation done on this tab. The “Pending Tasks” icon shows the status of last few such pending requests and their status. Also, once any such pending task is completed on the BIG-IP the UI reloads the data to display the latest AS3 information about partitions and applications. 
+
+For more details please check:  https://clouddocs.f5.com/f5-aci-servicecenter/latest/l4-l7.html#as3-async-task-processing
+
+------
+
+**Q. Why do I see warning signs against APIC Endpoints in View EPs table?**
+
+It is possible that AS3 plugin of BIG-IP is unable to sync the dynamic endpoints and create them as BIG-IP pool members due to various unsupported configurations such as:
+
+1. A duplicate node with the same IP as the endpoint is already present in another partition
+
+2. More than 60 endpoints have been added to this BIG-IP pool. AS3 currently supports only a maximum of 60 endpoints in an AS3 application pool.
+
+3. The node IP is a substring of another node IP.
+
+Please check if you have done any unsupported configurations. For details, please refer https://clouddocs.f5.com/f5-aci-servicecenter/latest/release-notes.html#dynamic-endpoint-attach-detach
```

Comment:
minus "Please"... Check if you...
minus "please" again, and plus "to"., ie:   "For details, refer to..."